pkg/querysharding/analyzer.go

```diff
@@ -38,6 +82,7 @@ func NewQueryAnalyzer() *QueryAnalyzer {
 //
 // The le label is excluded from sharding.
 func (a *QueryAnalyzer) Analyze(query string) (QueryAnalysis, error) {
+
```

Comment:
Why this line?

Reply:
Are you talking about line no. 82? If yes then, I can remove it.